qemu/tests/cfg/multi_host.cfg

```diff
@@ -37,15 +37,22 @@
                 #Time when start migration
                 - after_login_vm:
                     paused_after_start_vm = no
```

Comment:
Hi scui,
As we discussed, could you update it to a scenario at the same variant of reboot/stress.